### DISCUSSION TITLE
awesome updates, ty ben, Futurama beta

### DISCUSSION THREAD

**Discussion by nawnie** [TheLastBen/fast-stable-diffusion#949] [Show and tell]
wanted to say ty again, using the 1.5 model ive been able to do the entire cast* of futurama+ the style

here is a link to the session if anyone wants to try it out

https://drive.google.com/drive/folders/11V0zldUjIROGrMScstOXXXz_55boPd3B?usp=sharing

the issue with amy, is she got a caption of Futurama_wearing_pink_hoodie.

i recomend restore faces but eruler a at 30 works just fine havnt messed with others

this is 1.5

what it does
fry
bender
hermes
farnsworth
,futurama as a style tag
leela
nibbler
zoideberg.. kinda
and amy if you do a batch of 10 you may get one, running a new session fixing this issue now.
multiple settings are included just give a prompt a try
example

- leela standing next to fry

- Bender riding a horse

- idk just try em 🗡️ 

https://drive.google.com/drive/folders/11V0zldUjIROGrMScstOXXXz_55boPd3B?usp=sharing

nice to see the text encoder not force blend gender

slight nsfw due to show having nsfw, all images from the movies benders big game and benders big score, PG 13 source images at worse

**Comment by nawnie** [comment]
as it stands i believe a tad less text encoder used and a few more pictutres plus good captions will fix zoidberg, any chance we can get .txt files as captions again?

t encoder and steps where not watched lr was set to 4e-6 with a batch size of 4 on the t4 gpu 3ish hours?

**Comment by nawnie** [comment]
Fixed captions and full instance images, forgot i had attempted to add amy in before restarting so session doesnt contain all images used this one does with the fixed captions im using for this new run

https://drive.google.com/drive/folders/1a1sJefDaOnvPXNrcOJXjG5kb6ybb4fND?usp=share_link

**Comment by TheLastBen** [comment]
Impressive work !

**Reply by nawnie** [reply]
Ty, hernes wanted to be fry so much, that sweet spot on the encoder for multiple subjects is not very big.

**Comment by nawnie** [comment]
new additions worked well, trained text encoder for 1k and images for 4.5k, all characters look like themselves just... wobbly like edd ed and eddy so text encoding done for this many subjects at 1k DONT over train the text encoder, it seems to function fundamentally different then other repos,

gonna go for another 1k steps and check, do note my batch size is 3 so that's 3 x whatever steps im mentioning. tho this can be done as a free user so i do suggest it, max speed up is at a batch of 2 however.

**Comment by nawnie** [comment]
![01549-1565163146-leela, standing next to fry, futurama](https://user-images.githubusercontent.com/106923464/206866164-4a64994c-98e6-436d-8734-cf88561bcc35.png)

leela, standing next to fry, futurama

hope i'm not over saturating your comments but it seems people have been asking how to do this for awhile now

**Comment by nawnie** [comment]
less impressive, typing in will smith brings up conrad and tying jim carrey brings up fry, white male, black male, going to clip some pics from the head museum to see if i can fix that, but that may just be the limit of this text encoder, ill give it a try around 800 text steps and see what happens
